### PR TITLE
[Docs] Updating Voter doc for mainnet

### PR DESCRIPTION
**Pull request by rajkaramchedu**
Voter doc update. Let's land this, but know that I will likely update this doc once more before mainnet. 

cc: @movekevin

- - -
This change is [Reviewable](https://reviewable.io/reviews/aptos-labs/aptos-core/4879)

### CODE REVIEW

File: developer-docs-site/docs/nodes/validator-node/voter/index.md [hunk 1]
```diff
@@ -9,53 +9,24 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
...
-2. View the proposals. When you are ready to vote on a proposal, click on the proposal.
-3. Make sure you connected the wallet with your **voter** wallet account.
-4. Provide your **owner** account address and vote “For” or “Against”.
+1. Go to the [**Proposals section** of the Aptos Governance page](https://governance.aptosfoundation.org/).
+2. Next you should connect your wallet, but before you click on **CONNECT WALLET** (top-right):
+   1. Make sure you selected Mainnnet from the top-right drop-down menu.
+   2. Delete any previous versions of Aptos Wallet you have installed on Chrome.
+   3. **Install** the Petra (Aptos Wallet) extension on your Chrome browser by [following the instructions here](/guides/install-petra-wallet-extension).
+3. Click on **CONNECT WALLET** to connect your wallet to the Aptos Governance. 
+4. View the proposals. When you are ready to vote on a proposal, click on the proposal and vote.
```
Comment (msavir) [inline developer-docs-site/docs/nodes/validator-node/voter/index.md:29]: should we also tell them to make sure that mainnet is selected on the wallet?

**Review by msavir — APPROVED** [review]

File: developer-docs-site/docs/nodes/validator-node/voter/index.md (hunk 1 above)
Comment (rajkaramchedu) [reply developer-docs-site/docs/nodes/validator-node/voter/index.md:29]: Good idea. Best to say everything explicitly. I will make the change.

Comment (rajkaramchedu) [reply developer-docs-site/docs/nodes/validator-node/voter/index.md]: Just pushed the additional step. @msavir see if I got the step right? Thanks!

Comment (msavir) [reply developer-docs-site/docs/nodes/validator-node/voter/index.md]: LGTM

**Review by movekevin — APPROVED** [review]

**Review by sherry-x — APPROVED** [review]

**Pull request merged**